federation-js/package.json

```diff
@@ -17,7 +17,7 @@
   "author": "Apollo <packages@apollographql.com>",
   "license": "MIT",
   "engines": {
-    "node": ">=12.13.0 <18.0"
+    "node": "<18.13"
```

Comment:
I'd suggest instead dropping the `<` constraints in all the files, like we already have done on `main`. (Which is why https://github.com/apollographql/federation/pull/2216/files is much shorter!)